db/migrate/20201103134054_create_homepages.rb

```diff
@@ -0,0 +1,8 @@
+class CreateHomepages < ActiveRecord::Migration[6.0]
+  def change
+    create_table :homepages do |t|
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
You didn't actually add any columns to this table.  In the future you should avoid making tables like this.